src/modules/land_detector/LandDetector.cpp

```diff
@@ -96,6 +96,22 @@ void LandDetector::Run()
 
 	_update_topics();
 
+	if (!_dist_bottom_is_observable) {
```

Comment:
Once it's observable you never check again?

Reply:
@dagar Yes, the assumption is that the configuration will not change. Actually, I currently don't see any reason to not make it update.

Reply:
@dagar Hm, on a second thought I think I know why it was implemented like this. Consider the time when a range finder stops working, the ekf will then report dist bottom not to be observable. In that case you probably still want to assume that you have a range finder but the data is not valid and thus keep a high hysteresis. It's better than switching to low hysteresis and risking a false ground detection positive.
Thoughts welcome.